docker/docker-compose.yml

```diff
@@ -52,12 +54,12 @@ services:
     command: python3 /code/listenbrainz/manage.py runserver -h 0.0.0.0 -p 80 -d
     image: web
     volumes:
-      - ..:/code/listenbrainz
-      - ../credentials:/code/credentials
+      - ..:/code/listenbrainz:z
+      - ../credentials:/code/credentials:z
     environment:
       GOOGLE_APPLICATION_CREDENTIALS: '/code/credentials/bigquery-credentials.json'
     ports:
-      - "80:80"
+      - "8000:80"
```

Comment:
I think we should keep this at 80 and change the docs to 80 from 8000. Other MeB projects use 80 too and it is a good idea to keep as many similarities between projects as possible.

Reply:
@paramsingh What if you're running multiple projects in your environment? For the host machine, I thought port 8000 made more sense since it's an [unprivileged port](https://www.w3.org/Daemon/User/Installation/PrivilegedPorts.html), and you could use a web server (e.g. `nginx`) to do a rewrite rule for port 8000. In this way, port 8000 made more sense for both a development environment and running in production.

If you'd like, I can still change everything to port 80. But from my point of view, port 8000 makes more sense since port 80 is likely to conflict with other services / applications.

Reply:
You're right, 8000 does make more sense, I guess. I'm just hesitant because of other MeB projects using 80 (and a little, because my setup uses 80 :P). I'd be okay with 80 or 8000, but I guess we should make the port we choose here the standard for other MeB projects too.

Reply:
@mayhem, any opinions on this? :P

Reply:
I'd prefer consistency with other projects for now -- I agree with the reservations, but until we finally get some "best metabrainz docker practices" defined, lets go with 80.

Regardless of what we pick, people will complain about it. :)